src/qml/ChatPage.qml

```diff
@@ -59,18 +59,23 @@ Kirigami.Page {
 				Rectangle {
 					id: avatar
 
-					width: height
-					height: parent.height
+					width: 40
+					height: 40
+					radius: width*0.5
```

Comment:
#styling spaces between `*`
And maybe better move them into the first 'paragraph' to the id.